### PR TITLE
fix: inject correct MyInfo field titles in react

### LINKED ISSUE #4628: MyInfo builder edit field drawer copy changes
1. Drawer header names should be in Title Case (e.g. “Edit Date of Birth” - match question names from field list drawer if in doubt).
image

image

2. “Data source” should be in sentence case, same as “Verified for” and “Field details”
image

3. "Marriage Cert. No." in the edit field header, “Marriage certificate number” in the field details should be sentence case
image

4. Occupation and Name of Employer field details should say "Singpass" not "SingPass"
image

### PR DESCRIPTION
**Pull request by justynoh**
## Problem
Some MyInfo fields had copy edits to be made. This PR edits them.

Closes #4628 

EDIT (19 Oct 2022): Found a bug, there's backward incompatibility because the react app injects different myinfo field titles than angular does into the form definition. This PR patches this. Releasing at `v6.7.2`.

## Solution
Modified the edit drawer header to be identical to the field list drawer. Also updated some grammatical mistakes.

**Breaking Changes** 
- No - this PR is backwards compatible

### CODE REVIEW

File: shared/constants/field/myinfo/index.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export const types: MyInfoFieldBlock[] = [
   },
   {
     name: MyInfoAttribute.DateOfBirth,
-    value: 'Date of birth',
+    value: 'Date of Birth',
```
Comment (karrui) [inline shared/constants/field/myinfo/index.ts]: This is a shared file. Will this affect anything on the server? Is this truly backwards compatible?

**Comment by justynoh** [comment]
not a blocker for react, converting to draft first

**Comment by justynoh** [comment]
rebasing

File: shared/constants/field/myinfo/index.ts (hunk 1 above)
Comment (justynoh) [reply shared/constants/field/myinfo/index.ts]: yes and no :sob_smile:

**Review by karrui — APPROVED** [review]

**Comment by justynoh** [comment]
sorry the jest bump got muddled in the rebase :(

**Pull request merged**